static/window-template.tmpl

```diff
@@ -0,0 +1,333 @@
+        }
+
+        function info_online(url, data) {
+            // display info about current APK
+            fetch( url /*, options */)
+                .then((response) => response.text())
+                .then((html) => {
+                    document.getElementById("info").innerHTML = html || data;
+            })
+            .catch((error) => {
+                console.warn(error);
+            });
+        }
+
+
+        function info_inline(data){
+            document.getElementById("info").innerHTML = data
+        }
+
+        var ifram_dom
+
+        function childLoaded() { ifram_dom = childGetElementById('someid');}
+
+        function frame_online(url) {
+            window.frames["iframe"].location = url;
+        }
+
+        async function custom_site(vfs) {
+            // python and all filesystems are ready.
+            console.log(__FILE__, "custom_site Begin")
+
+            python.readline.history = []
+
+            //_rcp("current.py", "/data/data/org.python/assets/test.py")
+
+
+            frame_online(  Module.arguments[0]+".html" )
+
+
+
+    // PyConfig
+    // https://docs.python.org/dev/c-api/init_config.html
+
+
+
+            // your needs to the minimal pyconfig
+            python.PyConfig =  JSON.parse(`
+    {
+        "base_executable" : null,
+        "base_prefix" : null,
+        "buffered_stdio" : null,
+        "bytes_warning" : 0,
+        "warn_default_encoding" : 0,
+        "code_debug_ranges" : 1,
+        "check_hash_pycs_mode" : "default",
+        "configure_c_stdio" : 1,
+        "dev_mode" : -1,
+        "dump_refs" : 0,
+        "exec_prefix" : null,
+        "executable" : "${python.PyConfig.executable}",
+        "faulthandler" : 0,
+        "filesystem_encoding" : "utf-8",
+        "filesystem_errors" : "surrogatepass",
+        "use_hash_seed" : 1,
+        "hash_seed" : 1,
+        "home": null,
+        "import_time" : 0,
+        "inspect" : 1,
+        "install_signal_handlers" :0 ,
+        "interactive" : 1,
+        "isolated" : 1,
+        "legacy_windows_stdio":0,
+        "malloc_stats" : 0 ,
+        "platlibdir" : "lib",
+        "prefix" : "/data/data/org.python/assets/site-packages",
+        "ps1" : ">>> ",
+        "ps2" : "... "
+    }
+    `)
+
+            var runsite = `#!
+    import os,sys,json
+
+    PyConfig = json.loads("""${JSON.stringify(python.PyConfig)}""")
+
+    if os.path.isdir(PyConfig['prefix']):
+        sys.path.append(PyConfig['prefix'])
+        os.chdir(PyConfig['prefix'])
+
+    if os.path.isfile("/data/data/pythonrc.py"):
+        exec(open("/data/data/pythonrc.py").read(), globals(), globals())
+    # <- keep it here
+    `
+            if (python.motd)
+                runsite = runsite+ `print("_"*80);print("${python.motd}");print("_"*80);`
+
+
+            if (await _rcp("{{cookiecutter.cdn}}/pythonrc.py","/data/data/pythonrc.py")) {
+
+                // FIXME: sometimes runtime is called without overlayfs ready.

```

Comment:
you can remove that FIXME line it has been fixed.

Reply:
oo kk